### PR TITLE
new docker scripts

### LINKED ISSUE #120: Add yarn script to easily start up docker container for the server
# Requirements

- [ ] Add a yarn script. Probably `yarn start`? 

# Dependencies

- None

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #120

# What changed
- Add docker scripts
- Rename "docker:run" script

# Testing instructions
```sh
yarn docker:build
yarn docker:run
```

Visit http://localhost:49120/ and confirm you see Apollo server 🪐

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
         "prepare": "husky install",
         "test": "ts-mocha src/tests/**.ts",
         "dev": "yarn && ts-node-dev --transpile-only --no-notify --exit-child src/index.ts",
-        "docker:run": "ts-node src/index.ts",
```
Comment (ann-kilzer) [inline package.json:10]: Hey @ermish I like your script, but it made more sense to me to make `docker:run` connect to a docker command. So I called this one `prod`. Let me know your thoughts!

Comment (ermish) [reply package.json:10]: I like the direction! What if we took it one step further and the prod script runs all the docker scripts together to spin up the whole thing locally? 

so we'd have something like 
`yarn dev` -> run  locally without docker
`yarn prod` -> run docker commands all together `yarn docker:build && yarn docker:run`

Comment (ann-kilzer) [reply package.json:10]: We still need a name for the script that runs ts-node.... I'd like that to be `prod` or something similar

And I want to ensure we have scripts for running bare-metal.

So if you want another script, pick a name that is unclaimed

Comment (ann-kilzer) [reply package.json:10]: 'Serve' sounds too close to dev, some projects use it

Comment (theyokohamalife) [reply package.json:10]: `yarn staging`?

**Review by theyokohamalife — APPROVED** [review]
LGTM

**Review by ermish — APPROVED** [review]
Ship it! 🚀

**Pull request merged**